pkgs/development/libraries/wxSVG/default.nix

```diff
@@ -28,18 +30,17 @@ stdenv.mkDerivation rec {
     libexif
     pango
     wxGTK
-  ];
+  ] ++ lib.optional stdenv.isDarwin Cocoa;
 
   meta = with lib; {
-    homepage = "http://wxsvg.sourceforge.net/";
     description = "A SVG manipulation library built with wxWidgets";
     longDescription = ''
       wxSVG is C++ library to create, manipulate and render Scalable Vector
       Graphics (SVG) files with the wxWidgets toolkit.
     '';
-    license = with licenses; gpl2Plus;
-    maintainers = with maintainers; [ AndersonTorres ];
-    platforms = wxGTK.meta.platforms;
-    broken = stdenv.isDarwin;
+    homepage = "http://wxsvg.sourceforge.net/";
```

Comment:
When touching a package I'll usually reorder the meta attributes based on my understanding of what's best practice (if I notice).

Reply:
Well, I personally prefer `homepage` to be the first `meta`, because the homepage is the real source of information about a project.
After that, short `description` and `longDescription`.

And, after them, the other attrs in more or less alphabetical order.

There is no reason to change it besides aesthetic.

Reply:
👍